playbook/app/pb_kits/playbook/pb_bread_crumbs/_bread_crumb_item.tsx

```diff
@@ -1,55 +1,69 @@
-import React from 'react'
-import classnames from 'classnames'
-import { globalProps, domSafeProps } from '../utilities/globalProps'
+import React from "react"
+import classnames from "classnames"
+import { globalProps, domSafeProps } from "../utilities/globalProps"
 
 import {
   buildAriaProps,
   buildCss,
   buildDataProps,
-  buildHtmlProps } from '../utilities/props'
+  buildHtmlProps,
+} from "../utilities/props"
 
 type BreadCrumbItemProps = {
-  aria?: {[key: string]: string},
-  className?: string,
-  data?: {[key: string]: string},
-  htmlOptions?: {[key: string]: string | number | boolean | (() => void)},
-  id?: string,
-  component?: "a" | "span",
-  [x:string]: any;
+  aria?: { [key: string]: string }
+  children?: React.ReactNode
+  className?: string
+  component?: "a" | "span"
+  data?: { [key: string]: string }
+  htmlOptions?: { [key: string]: string | number | boolean | (() => void) }
+  id?: string
+  [x: string]: any
 }
 
 const BreadCrumbItem = (props: BreadCrumbItemProps): React.ReactElement => {
   const {
     aria = {},
+    children,
     className,
+    component = "a",
     data = {},
     htmlOptions = {},
     id,
-    component = 'a',
     ...rest
   } = props
+
   const ariaProps = buildAriaProps(aria)
   const dataProps = buildDataProps(data)
   const htmlProps = buildHtmlProps(htmlOptions)
-  const Component = component || 'span';
+  const Component = component || "span"
   const css = classnames(
-    buildCss('pb_bread_crumb_item_kit'),
+    buildCss("pb_bread_crumb_item_kit"),
     globalProps(props),
     className
   )
-  
+
+  const processedChildren = React.Children.map(children, (child) => {
+    if (React.isValidElement<{ className?: string }>(child)) {
+      return React.cloneElement(child, {
+        className: classnames(child.props.className, globalProps(props)),
```

Comment:
This is close, but we need to only spread the `dark` prop... not all of the globalProps. Otherwise, I think when a dev adds... let's say... `margin="sm"` it would add that `margin_sm` classname to all children elements, which would be bad. Let me confirm this by testing your branch locally.

Reply:
yeah let me know happy to explore

Reply:
i see what you are saying. i reverted back and update react example to include props for Title and Icons